core/app/models/spree/google_export_option.rb

```diff
@@ -1,7 +1,11 @@
 module Spree
   class GoogleExportOption < Base
+    belongs_to :spree_store
+
+    validates :spree_store_id, presence: true
```

Comment:
```suggestion
    validates :store, presence: true
```
similar to: https://github.com/spree/spree/blob/main/core/app/models/spree/order.rb#L165